pkg/flink/CONTRIBUTING.md

```diff
@@ -12,13 +12,20 @@ go tool cover -html=coverage.out -o coverage.html
 You can then open the coverage.html file from the repository root in your browser to see the coverage report for each file.
 
 ### Generating Mocks
-We have generated mocked as we need them. We have a make target for controllers, store and client. You can look at the mock_generator.go file as an example if you need to mock other things.
+We generate mocks for interfaces as we need them. If you update an interface, you might have to generate mocks again. Make sure you have mockgen installed:
 
+```
+go install go.uber.org/mock/mockgen@latest
+```
+
+Generate mocks:
 ```
 go generate pkg/flink/test/mock/mock_generator.go
 ```
 
-More info on how to generate mocks: https://github.com/golang/mock
+If you need to generate mocks for a new interface, take a look at the mock_generator.go and change it accordingly.
```

Comment:
What's the reason for switching from the golang version to the uber version?

Reply:
I was hoping to standardize the golang version across the CLI

Reply:
golang one is no longer maintained, the go.uber.org fork is the one where the development is currently taking place